clio-client/src/test/resources/testdata/gvcfBadmetadata

```diff
@@ -0,0 +1 @@
+you can't parse this
```

Comment:
Could we just have a generic "bad metadata" file that all the parsing tests could use, instead of having separate ones for wgs-ubam, gvcf, etc.?